File: go/udp_client.go
package main;

import "net"
import "log"


func main() {
    connection,err:=net.Dial("udp6", "[::1]:1234")
    if err!=nil{
        log.Fatalln("Error (net.Dial):", err)
    }
    defer connection.Close()

    n,err:=connection.Write([]byte("Hello(udp) from (go)client"))
    if err!=nil{
        log.Fatalln("Error (connection.Write):", err)
    }

    log.Println("Data sent:", n)
}

File: rust/udp_client.rs
use std::net::UdpSocket;


fn main() {
    let socket=UdpSocket::bind("[::]:0").expect("UdpSocket::bind");

    match socket.send_to("Hello(udp) from (rust)client".as_bytes(), "[::1]:1234") {
        Err(e) => println!("Error (connection.Write): {}", e),
        Ok(n) => println!("Data sent: {}", n),
    }
}